src/redux/actions/location.action.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable indent */
+/* eslint-disable key-spacing */
+/* eslint-disable import/prefer-default-export */
+
+import { GETLOCATIONS } from '../actionTypes/actionTypes';
```

Comment:
The naming of your files is different from what others are using. It might cause problems while rebasing.
We have used: types, action, and reducers as folders in redux.